AdminApi/test/AdminApi.Tests.Integration/Features/IdentityDeletionProcess/AuditLogs/GET.feature

```diff
@@ -0,0 +1,11 @@
+﻿@Integration
+Feature: GET Identities/{identityAddress}/DeletionProcesses/AuditLogs
+
+User requests the Audit Logs pertaining to all Deletion Processes of an Identity
+
+Scenario: Requesting Audit Logs of Deletion Processes of Identity
+	Given an Identity i
+	And an active deletion process for Identity i exists
```

Comment:
1. The audit log entries of a cancelled deletion process should be returned as well.
2. The audit log entries of a rejected deletion process should be returned as well.
3. The audit log entries of a completed deletion process should be returned as well.

=> I would create a single identity which has deletion processes in each of the above statuses.

Reply:
I can cover active and cancelled, but how would I go about having completed and reject deletion processes in the Admin Api integration tests? Can they not only be completed or cancelled through the Consumer Api?

Reply:
I hope that you've done a manual test with deletion processes in the above statuses.